src/components/dialogs/limits/operational-limits-group-tab-label.tsx

```diff
@@ -52,6 +62,9 @@ export function OperationalLimitsGroupTabLabel({
                         ''
                     )}
                 </Stack>
+                <FormHelperText error={!!permanentLimitErrorMessage}>
+                    {permanentLimitErrorMessage ? <ErrorOutlineOutlinedIcon /> : ''}
+                </FormHelperText>
```

Comment:
I think this is better to do it with a single condition instead of 2 : 

```suggestion
                {permanentLimitErrorMessage && (
                    <FormHelperText error>
                        <ErrorOutlineOutlinedIcon />
                    </FormHelperText>
                )}
```